cmd/users/main.go

```diff
@@ -36,39 +38,41 @@ import (
 )
 
 const (
-	defLogLevel      = "error"
-	defDBHost        = "localhost"
-	defDBPort        = "5432"
-	defDBUser        = "mainflux"
-	defDBPass        = "mainflux"
-	defDBName        = "users"
-	defDBSSLMode     = "disable"
-	defDBSSLCert     = ""
-	defDBSSLKey      = ""
-	defDBSSLRootCert = ""
-	defHTTPPort      = "8180"
-	defGRPCPort      = "8181"
-	defSecret        = "users"
-	defServerCert    = ""
-	defServerKey     = ""
-	defJaegerURL     = ""
-
-	envLogLevel      = "MF_USERS_LOG_LEVEL"
-	envDBHost        = "MF_USERS_DB_HOST"
-	envDBPort        = "MF_USERS_DB_PORT"
-	envDBUser        = "MF_USERS_DB_USER"
-	envDBPass        = "MF_USERS_DB_PASS"
-	envDBName        = "MF_USERS_DB"
-	envDBSSLMode     = "MF_USERS_DB_SSL_MODE"
-	envDBSSLCert     = "MF_USERS_DB_SSL_CERT"
-	envDBSSLKey      = "MF_USERS_DB_SSL_KEY"
-	envDBSSLRootCert = "MF_USERS_DB_SSL_ROOT_CERT"
-	envHTTPPort      = "MF_USERS_HTTP_PORT"
-	envGRPCPort      = "MF_USERS_GRPC_PORT"
-	envSecret        = "MF_USERS_SECRET"
-	envServerCert    = "MF_USERS_SERVER_CERT"
-	envServerKey     = "MF_USERS_SERVER_KEY"
-	envJaegerURL     = "MF_JAEGER_URL"
+	defLogLevel           = "error"
+	defDBHost             = "localhost"
+	defDBPort             = "5432"
+	defDBUser             = "mainflux"
+	defDBPass             = "mainflux"
+	defDBName             = "users"
+	defDBSSLMode          = "disable"
+	defDBSSLCert          = ""
+	defDBSSLKey           = ""
+	defDBSSLRootCert      = ""
+	defHTTPPort           = "8180"
+	defGRPCPort           = "8181"
+	defSecret             = "users"
+	defServerCert         = ""
+	defServerKey          = ""
+	defJaegerURL          = ""
+	defTokenResetEndpoint = "/password/reset"
```

Comment:
Why is this configurable?

Reply:
https://github.com/mainflux/mainflux/pull/873#discussion_r334329934